src/dask_awkward/lib/core.py

```diff
@@ -570,6 +573,15 @@ def f(*args):
         args = tuple(
             ak.Array(arg.content) if isinstance(arg, MaybeNone) else arg for arg in args
         )
+        args = tuple(
```

Comment:
Here we account for getting the low-level typetracer through during column optimisation. This is the questionable part.